### PR TITLE
Remove eris-logger references from eris-db server

### PR DESCRIPTION
**Pull request by silasdavis**
Little incremental refactor stripping out eris-logger

### CODE REVIEW

**Comment by silasdavis** [comment]
@VoR0220 this change entails no tests. It's just changing existing logging to use new framework. There are some tests in the logging framework itself which is already merged to this feature branch.

**Comment by VoR0220** [comment]
@silasdavis what I mean is, are all the chains server tests cleaned of the eris-logger?

**Comment by silasdavis** [comment]
Can someone press the button

File: event/event_cache.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func reap(es *EventSubscriptions) {
 	defer es.mtx.Unlock()
 	for id, sub := range es.subs {
 		if time.Since(sub.ts) > reaperThreshold {
-			// Seems like Go is ok with this..
+			// Seems like Go is ok with evts..
```
Comment (benjaminbollen) [inline event/event_cache.go]: You were a bit quick with the replace alls for `this` -> `evts`

Comment (silasdavis) [inline event/event_cache.go]: fixed in current working directory

**Pull request merged**